README.zh-CN.md

```diff
@@ -61,23 +61,198 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
+```
+tar -xvzf apache-eventmesh-1.10.0-bin.tar.gz
+cd apache-eventmesh-1.10.0
+```
 
-[第1步: 使用Docker部署EventMesh Store](https://eventmesh.apache.org/docs/instruction/store-with-docker)
+#### 2. 运行  
 
-[第2步: 使用Docker启动EventMesh Runtime](https://eventmesh.apache.org/docs/instruction/runtime-with-docker)
+编辑`eventmesh.properties`以更改EventMesh Runtime的配置（如 TCP 端口、客户端黑名单）。
+```
+vim conf/eventmesh.properties
+```
 
-[第3步: 运行示例](https://eventmesh.apache.org/docs/instruction/demo)
+指定事件存储为 RocketMQ(默认为standalone)：
+```
+# storage plugin
+eventMesh.storage.plugin.type=rocketmq
+```
+
+执行`start.sh`脚本启动EventMesh Runtime服务器。
+```
+bash bin/start.sh
+```
+如果看到`EventMeshTCPServer[port=10000] started....`, 则说明设置成功。
+
+查看输出日志:
+```
+cd /root/apache-eventmesh-1.10.0/logs
+tail -f eventmesh.out
+```
+
+停止:
+```
+bash bin/stop.sh
+```
+
+### 在 Docker 中运行 EventMesh Runtime  
+
+#### 1.获取EventMesh镜像
+
+首先，你可以打开一个命令行，并且使用下面的`pull`命令从[Docker Hub](https://hub.docker.com)中下载最新发布的[EventMesh](https://hub.docker.com/r/apache/eventmesh)。
+```
+sudo docker pull apache/eventmesh:v1.10.0
+```
+
+您可以使用以下命令列出并查看本地已有的镜像。
+```
+$ sudo docker images
+REPOSITORY         TAG       IMAGE ID       CREATED      SIZE
+apache/eventmesh   latest    f32f9e5e4694   2 days ago   917MB
+```
+
+#### 2.创建配置文件:
+
+在根据EventMesh镜像运行对应容器之前，你需要创建两个配置文件，分别是:`eventmesh.properties`和`rocketmq-client.properties`。  
+首先，你需要使用下面的命令创建这两个文件。
+```
+sudo mkdir -p /data/eventmesh/rocketmq/conf
+cd /data/eventmesh/rocketmq/conf
+sudo touch eventmesh.properties
+sudo touch rocketmq-client.properties
+```
+
+#### 3.配置`eventmesh.properties`
+
+这个配置文件中包含 EventMesh 运行时环境和集成进来的其他插件所需的参数。  
+使用下面的`vim`命令编辑`eventmesh.properties`。
+```
+sudo vim eventmesh.properties
+```
+你可以直接将 GitHub 仓库中的对应[配置文件](https://github.com/apache/eventmesh/blob/1.10.0-prepare/eventmesh-runtime/conf/eventmesh.properties)中的内容复制过来。  
+
+指定事件存储为 RocketMQ(默认为standalone)：
+```
+# storage plugin
+eventMesh.storage.plugin.type=rocketmq
+```
+
+请检查配置文件里的默认端口是否已被占用，如果被占用请修改成未被占用的端口:
+
+| 属性                                 | 默认值     | 备注                           |   
+|------------------------------------|---------|------------------------------|  
+| `eventMesh.server.http.port`       | `10105` | `EventMesh http server port` |  
+| `eventMesh.server.tcp.port`        | `10000` | `EventMesh tcp server port`  | 
+| `eventMesh.server.grpc.port`       | `10205` | `EventMesh grpc server port` | 
+| `eventMesh.server.admin.http.port` | `10106` | `HTTP management port`       | 
+
+#### 4.配置`rocketmq-client.properties`
+
+这个配置文件中包含 EventMesh 运行时环境和集成进来的其他插件所需的参数。
+
+使用下面的`vim`命令编辑`rocketmq-client.properties`。
+```
+sudo vim rocketmq-client.properties
+```
+你可以直接将GitHub仓库中的对应[配置文件](https://github.com/apache/eventmesh/blob/1.10.0-prepare/eventmesh-storage-plugin/eventmesh-storage-rocketmq/src/main/resources/rocketmq-client.properties)中的内容复制过来
+> 请注意，如果您正在运行的 namesetver 地址不是配置文件中的默认值，请将其修改为实际正在运行的nameserver地址。
+
```

Comment:
Please use the latest version Docker Deployment document in https://eventmesh.apache.org/zh/docs/instruction/runtime-with-docker (This page is updated today). The old version doc has many mistakes and its commands are impractical.